covid_moonshot/analysis/plots.py

```diff
@@ -0,0 +1,436 @@
+from contextlib import contextmanager
+from functools import partial
+import os
+import logging
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+from simtk.openmm import unit
+from openmmtools.constants import kB
+from typing import List, Optional, Tuple
+from ..core import Binding, PhaseAnalysis, RunAnalysis, Work
+
+TEMPERATURE = 300.0 * unit.kelvin
+KT = kB * TEMPERATURE
+
+
+def plot_work_distribution(
+    ax: plt.Axes, forward_works: List[float], reverse_works: List[float]
+) -> None:
+    """
+    Plot a single work distribution
+
+    Parameters
+    ----------
+    ax : AxesSubplot
+       Axes on which to draw the plot
+    forward_works : list of float
```

Comment:
Apologies for just realizing this: Would be good to document units for work or energy values if they are floats! You can just say "(in kT)" for all these.

Reply:
Cool, I can add this. My convention so far is to assume kT and specify only when we differ from that, but explicit is better than implicit!

Reply:
Will be cool to transition to `pint` or something similar!

Reply:
Annotated docstrings with `(in kT)` for now: 7b59b1e